File: Client/src/app/models/user.model.ts
export class UserModel
{
    private _Id: string;
    private _UserName: string;
    private _SecondName: string;
    private _Phone: string;
    private _EMail: string;

    constructor( id: string = "",
                 userName: string = "",
                 secondName: string = "",
                 phone: string = "",
                 email: string = "" ) {

        this.UserName = userName;
        this.Id = id;
        this.SecondName = secondName;
        this.Phone = phone;
        this.EMail = email;            
    }

    get UserName(): string
    {
        return this._UserName;
    }

    set UserName( value: string)
    {
        this._UserName = value;
    }

    get SecondName(): string
    {
        return this._SecondName;    
    }

    set SecondName( value: string )
    {
        this._SecondName = value;    
    }

    get Phone(): string
    {
        return this._Phone;    
    }

    set Phone( value: string )
    {
        this._Phone = value;    
    }

    get EMail(): string
    {
        return this._EMail;    
    }

    set EMail( value: string )
    {
        this._EMail = value;    
    }

    get Id(): string
    {
        return this._Id;    
    }

    set Id( value: string )
    {
        this._Id = value;    
    }
}

File: Service/Models/UserModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace JobMApi.Models
{
    /// <summary>
    /// 
    /// </summary>
    public class UserModel
    {
        /// <summary>
        /// 
        /// </summary>
        [Required]
        [MaxLength( 128 )]
        [JsonProperty( PropertyName = "id" )]
        public String Id
        {
            get;
            set;
        }

        /// <summary>
        /// 
        /// </summary>
        [Required]
        [JsonProperty( PropertyName = "user_name")]
        [StringLength( 100, ErrorMessage = "The {0} must be at least {2} characters long.", MinimumLength = 3 )]
        public String UserName
        {
            get;
            set;
        }

        /// <summary>
        /// 
        /// </summary>
        [Required]
        [MaxLength(256)]
        [JsonProperty( PropertyName = "second_name" )]
        public String SecondName
        {
            get;
            set;
        }

        /// <summary>
        /// 
        /// </summary>
        [Required]
        [MaxLength(256)]
        [JsonProperty( PropertyName = "email" )]
        public String EMail 
        {
            get;
            set;
        }

        /// <summary>
        /// 
        /// </summary>
        [Required]
        [MaxLength( 256 )]
        [JsonProperty( PropertyName = "phone_number" )]
        public String Phone
        {
            get;
            set;
        }


        /// <summary>
        /// Ctor default
        /// </summary>
        public UserModel() 
        {
        }
    }
}